land.md

```diff
@@ -185,7 +185,23 @@ This section summarizes some of the land-based policy options available in GCAM.
 
 ### Protecting Lands
 
-With this policy, we can set aside some land, removing it from economic competition. This will result in that land area being fixed across time and any land expansion/contraction will not affect this area. The default in GCAM is to protect 90% of all non-commercial ecosystems. This protection level is specified in the GCAM data system; see `aglu.PROTECT_LAND_FRACT` in [constants.R](https://github.com/JGCRI/gcam-core/blob/master/input/gcamdata/R/constants.R).
+With this policy, we can set aside some land, removing it from economic competition. This will result in that land area being fixed across time and any land expansion/contraction will not affect this area. 
+
+The default in GCAM is to protect 90% of all non-commercial ecosystems. This protection level is specified in the GCAM data system; see `aglu.PROTECTION_DATA_SOURCE_DEFAULT` in [constants.R](https://github.com/JGCRI/gcam-core/blob/master/input/gcamdata/R/constants.R#L344). Users can define a custom percentage of protected lands using the parameter `aglu.PROTECT_DEFAULT`.
```

Comment:
Note, these links won't point to the right place until all `gcam-core` changes are merged and pushed out to Github.  But of course this PR is going to the `gcam-v5.5` holding brach so that is ok.